scheduler/src/cook/config.clj

```diff
@@ -199,9 +199,11 @@
                                      ((util/lazy-load-var 'cook.impersonation/create-impersonation-middleware) impersonators)
                                      {:json-value "config-impersonation"})))
      :rate-limit (fnk [[:config {rate-limit nil}]]
-                   (let [{:keys [user-limit-per-m]
+                   (let [{:keys [user-limit-per-m rate-limit-expire-min]
```

Comment:
Nitpick: lexicographic order

Reply:
And another in creating the dictionary a few lines lower.